app/renderer/new_api/createDataRequest.ts

```diff
@@ -0,0 +1,18 @@
+import { ApiClient, ApiMethod, MethodArgs } from "./"
+import { CreateDataRequestParams, GenericAPIError, DataRequest } from "../../common/runtimeTypes/wallet"
+import { asRuntimeType } from "app/common/runtimeTypes"
+
+export function createDataRequest(client: ApiClient<ApiMethod, MethodArgs>, params: CreateDataRequestParams): Promise<any> {
```

Comment:
Can we replace any with JsonRT?
```suggestion
export function createDataRequest(client: ApiClient<ApiMethod, MethodArgs>, params: CreateDataRequestParams): Promise<JsonRT> {
```